homeassistant/components/freebox/button.py

```diff
@@ -35,10 +35,18 @@ class FreeboxButtonEntityDescription(
 BUTTON_DESCRIPTIONS: tuple[FreeboxButtonEntityDescription, ...] = (
     FreeboxButtonEntityDescription(
         key="reboot",
-        name="Reboot Freebox",
+        name="Reboot",
+        has_entity_name=True,
```

Comment:
Let's keep this out until follow-up PR.
```suggestion
        name="Reboot Freebox",
```